src/sentry/templatetags/sentry_helpers.py

```diff
@@ -11,7 +11,7 @@
 from django.utils.html import escape
 from django.utils.safestring import mark_safe
 from django.utils.translation import ugettext as _
-from pkg_resources import parse_version as Version
+from packaging.version import parse as Version
```

Comment:
Can we standardize on `from packaging.version import Version`, or whatever you prefer? I assume `Version(s)` is basically the same as `parse(s)`.

Reply:
`parse` is try: Version except LegacyVersion` -- I think I'll standardize on `parse_version` when it's just `parse` and `Version` otherwise